_common

```diff
@@ -131,7 +131,9 @@ handle_unknown() {
         group_description=$(echo "$group_data" | runjq -r '.[0].description' ) || true
         group_mailto=$(echo "$group_description" | perl -n -wE'm/.*MAILTO: (\S+).*/ and say $1' | head -1)
         group_mailto=${group_mailto:-$notification_address}
-        if [[ -n "$group_mailto" ]]; then
+        # Avoid sending notifications for jobs that were restarted
+        clone_id="$(echo "$job_data" | runjq -r '.job.clone_id')"
```

Comment:
```suggestion
        clone_id=$(echo "$job_data" | runjq -r '.job.clone_id')
```

Reply:
Did not notice the double double quotes there. It works though:
```
$ job_data='{"job": {"name": "foo", "result": "failed"}}'
$ A="$(echo "$job_data" | jq -r '.job.clone_id')"
$ echo $A
null
$
```
However this got merged right after your approval. Should I submit a new PR to get rid of the outermost double quotes, or leave as is?

Reply:
You can leave it as-is. I've just mentioned it because quotes are not necessary when assigning variables like this. (They to no harm of course.)